pkgs/tools/audio/mpd-mpris/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "mpd-mpris";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "natsukagami";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19sz1ykdzradhpdwdvvxh9scp5sv2i072qircs0q4374cdgccfbb";
+  };
+
+  modSha256 = "1a95kfy8w952269x4llbl0afyxr5fjkg30mxsn81zdh5wr8gabwh";
+
+  subPackages = [ "cmd/${pname}" ];
+
+  postInstall = ''
+    substituteInPlace mpd-mpris.service \
+      --replace /usr/bin $out/bin
+    mkdir -p $out/lib/systemd/user
+    cp mpd-mpris.service $out/lib/systemd/user
```

Comment:
Shouldn't this be a NixOS module instead? cc @samueldr @cleverca22 @Infinisil.

Reply:
Nah this is fine, NixOS supports service files in packages with `systemd.packages`

Reply:
@Infinisil could you explain why I'm not able to enable the systemd service with `systemctl enable` when the package is installed to my user environment? I've wrote about it in [my first comment](https://github.com/NixOS/nixpkgs/pull/65340#issue-300804704).

Reply:
Ah, it's a user service, missed that. I don't think I've ever seen those being used as you described with nix-env, so maybe nobody ever thought of making that work.

It also doesn't seem like [home-manager](https://github.com/rycee/home-manager) supports running service files from packages, but that's definitely a place where you could add support for that without much trouble.

Reply:
So you are saying in general that systemd's user services search path just doesn't include `~/.nix-profile/lib/systemd/user/`? That's rather unfortunate.. In any case, I guess it would be find to leave it as ease, unless you wish to add a user module service like `services.keybase` for example..

Reply:
I haven't researched a lot, it might end up being pretty easy to add that to systemd's paths

Reply:
TIL after reading [this](https://unix.stackexchange.com/questions/224992/where-do-i-put-my-systemd-unit-file) that if I `ln -s ~/.nix-profile/lib/systemd ~/.local/share/systemd`, `systemctl enable` works.. However, trying out the following:

```
export SYSTEMD_UNIT_PATH="$HOME/.nix-profile/lib/systemd/user:"
systemctl --user enable mpd-mpris
```

Without the symlink, didn't work..